spec/helpers/hq/time_helper_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Hq::TimeHelper, type: :helper do
+  it 'formats a date' do
+    expect(Hq::TimeHelper.date_to_string(Date.current)).to be_a_kind_of String
+  end
+
+  it "doesn't raise exception on nil" do
```

Comment:
yes the only remaining question is what happens when someone calls it with something that is not a date?
Do we want it to blow up or do we want to return a Null Date?
If we want to return a null date in such circumstances do we want the message to be an argument to null date instantiation with a default ?